DragaliaAPI/Services/Game/LoadService.cs

```diff
@@ -45,6 +47,17 @@ public async Task<LoadIndexData> BuildIndexData()
 
         // TODO/NOTE: special shop purchase list is not set here. maybe change once that fully works?
 
+        int maxPartyPower = 0;
+
+        foreach (DbParty party in savefile.PartyList)
+        {
+            int currentPartyPower = await partyPowerService.CalculatePartyPower(party, bonusList);
+            if (currentPartyPower > maxPartyPower)
+                maxPartyPower = currentPartyPower;
+        }
```

Comment:
How expensive is this?

If I'm reading the service right each party power calculation reads up to 4 units, and could perform 6 queries per unit. Or is it reasonably fast because all the data is in the change tracker?